packages/FAQ/docs/creating-a-smart-invoice.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Creating a Smart Invoice
+--- 
+
+## Step 1:  Project Details 
+
+Here you will enter your project’s details in sequential order: 
+
+1. Project Name or ID
+2. Link to Project Agreement
+3. Project Description
+4. Project Start / End / Safety Valve (DATE)
+
+<h4><details>
+<summary> What is Safety Valve and Why is it Important?  </summary>    
+<p>&nbsp;</p>
+
+#### Safety Valve is a built in function that allows the Client to withdraw funds from the contract on the date specified in the date entry field.  If either party goes unresponsive or bails on the project, this is the date at which the client is free to withdrawal all funds deposited into Smart Invoice.
+
+</details></h4>
+
+## Step 2:  Payment Details 
+
+In this step you will enter the following payment details: 
+
+1. Client Address 
```

Comment:
Same as above - can we give any suggestions or further explanations for what these are? Otherwise we're just repeating the app itself in text form.